openapi.yaml

```diff
@@ -0,0 +1,89 @@
+openapi: 3.0.3
+info:
+  title: Klaw - OpenAPI
+  version: 1.0.0
+  description: >
+    This specification is still a work in progess and is not yet implemented in any API.
+  contact:
+    email: info@klaw-project.io
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+externalDocs:
+  description: Klaw documentation
+  url: https://www.klaw-project.io/docs
+tags:
+  - name: User
+    description: User API endpoints
+paths:
+  /user/authentication:
```

Comment:
What do you think we if would change this into `/user/authenticate`.

Reply:
Looks better `/user/authenticate`